extensions/amp-story/1.0/amp-story.css

```diff
@@ -224,7 +232,7 @@ amp-story-page:not(:first-of-type):not([distance]):not([active]) {
   transform: translateY(1000vh) !important;
 }
 
-amp-story-page:not([active]) {
+amp-story.i-amphtml-story-loaded amp-story-page:not([active]) {
```

Comment:
Why do you need this change?

Reply:
As seen offline, let's revisit this and only rely on `[distance]` and `[active]` attr